### PR TITLE
enh(release): Add Centreon Web 20.10.7 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x (master)

### CODE REVIEW

**Review by mdebauchez — CHANGES_REQUESTED** [review]
In english and french version : 
Ligns 46 and 47 are the same

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,51 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Platform Topology] API: remote / poller to central with proxy
+- [Platform Topology] Remove all topology when Remote => Central Conversion
+- [Platform Topology] Script: Unable to register a remote on a Central
+- [Platform Topology] Unable to replace 127.0.0.1 by real IP in poller form when already saved in platform_topology
+- [Purge] Script can't drop several partitions
+- [Resources Status] Be able to sort by Parent name in listing
+- [Resources Status] Manage notes on resources for Notes URL
+- [Resources Status] Missing french translations
+- [Resources Status] Service link to multiple hosts or service by hostgroup won't be listed
+- [Resources Status] Service link to multiple hosts or service by hostgroup won't be listed
```
Comment (tuntoja) [inline en/releases/centreon-core.md:47]:
```suggestion
```

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,51 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Platform Topology] API: remote / poller to central with proxy
+- [Platform Topology] Remove all topology when Remote => Central Conversion
+- [Platform Topology] Script: Unable to register a remote on a Central
+- [Platform Topology] Unable to replace 127.0.0.1 by real IP in poller form when already saved in platform_topology
+- [Purge] Script can't drop several partitions
+- [Resources Status] Be able to sort by Parent name in listing
+- [Resources Status] Manage notes on resources for Notes URL
+- [Resources Status] Missing french translations
+- [Resources Status] Service link to multiple hosts or service by hostgroup won't be listed
+- [Resources Status] Service link to multiple hosts or service by hostgroup won't be listed
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:47]:
```suggestion
```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,50 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.7
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`May 7, 2021`

```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,50 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.7
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`7 mai 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**